### PR TITLE
trigger a violation when node is discovered (added)

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
Allow to trigger a violation when a node is discovered (added)

# Impacts
node_add flow

# Delete branch after merge
YES

# NEWS file entries
## Enhancements
* Allow to trigger a violation and its actions when a node is discovered

### CODE REVIEW

File: conf/pfqueue.conf.example [hunk 1]
```diff
@@ -18,7 +18,7 @@ redis_every=100
 #
 [queue general]
 workers=4
-has_delayed_queue=disabled
+has_delayed_queue=enabled
```
Comment (jrouzierinverse) [inline conf/pfqueue.conf.example]: Should be in conf/pfqueue.conf.defaults

**Review by jrouzierinverse — CHANGES_REQUESTED** [review]
Looks good outside of the configuration issue.

**Comment by fdurand** [comment]
Except the comment of James i am ok to merge.

**Comment by julsemaan** [comment]
Fixed according to james comment

**Pull request merged**